### DISCUSSION TITLE
Use it with Angular web app

### DISCUSSION THREAD

**Discussion by TarakKar** [jlongster/absurd-sql#61] [Q&A]
Can anyone suggest how I implement this in the angular web app?  I am not able to import the modules in typescript.

Thanks.

**Comment by quolpr** [comment]
Absurd-sql is a pretty low level API, but you can try to use Kikko https://kikko-doc.netlify.app/core/usage . It gives some nice abstraction to use any SQLite backend out-of-the box, and It has `absurd-sql` backend support. You can even to subscribe to table changes via `listenQueries`
